packages/backend/src/utils.ts

```diff
@@ -27,3 +30,41 @@ export const isDbPinnedChart = (
     data: DbPinnedChart | DbPinnedDashboard,
 ): data is DbPinnedChart =>
     'saved_chart_uuid' in data && !!data.saved_chart_uuid;
+
+export const wrapSentryTransaction = async (
```

Comment:
Wrap method to add spans to sentry 

https://lightdash.sentry.io/performance/lightdash:0920559162d24063a216e9cf13cc4528/?project=5959292&query=%2Fapi%2Fv1%2Fprojects%2F963e0810-ecac-47dc-b170-444c84ea0601%2Fexplores%2F%3AexploreId%2FdownloadCsv&referrer=performance-transaction-summary&statsPeriod=24h&transaction=POST+%2Fapi%2Fv1%2Fprojects%2F963e0810-ecac-47dc-b170-444c84ea0601%2Fexplores%2F%3AexploreId%2FdownloadCsv&unselectedSeries=p100%28%29

![image](https://user-images.githubusercontent.com/1983672/218103744-3af78983-c531-40cd-8675-186d7e0b9bca.png)

Reply:
This wrapper is really useful 👨‍🍳 💋